systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -226,4 +226,23 @@ public static JsonArray expectedServiceDiscoveryInfo(String plainAuth, String tl
         jsonArray.add(expectedServiceDiscoveryInfo(9093, "kafka", tlsAuth).getValue(0));
         return jsonArray;
     }
+
+    public static boolean checkLogForJSONFormat(Map<String, String> pods, String containerName) {
+        List<String> podNames = new ArrayList<>(pods.keySet());
+        boolean isJSON = false;
+
+        for (String podName : podNames) {
```

Comment:
There's no need to create `podNames` AFAICS. You just want to iterate over all the pod names, which you can do using `for (String podName : pods.keySet()) { ... }`.

Reply:
Thanks for nit, done :)